lib/pbench/test/functional/agent/cli/commands/test_config.py

```diff
@@ -4,6 +4,7 @@
 
 
 def test_pbench_run(monkeypatch, tmpdir, agent_config, pbench_run):  # noqa F811
+    """Use pbench_run value from pbench-config"""
```

Comment:
Maybe "from the `pbench-agent.cfg` file? (Or even just "from the config file"?) In this context we're not using the `pbench-config` command so this could be a bit confusing.